examples/iam-assumable-roles-with-saml/main.tf

```diff
@@ -58,3 +58,18 @@ module "iam_assumable_roles_with_saml_custom" {
 
   provider_id = aws_iam_saml_provider.idp_saml.id
 }
+
+################################################
+# IAM assumable roles with SAML with self assume
+################################################
+module "iam_assumable_roles_with_saml_with_self_assume" {
```

Comment:
ALL examples should be executable right away, as described in the README - `terraform init -upgrade && terraform apply -auto-approve && terraform destroy -auto-approve`.

Currently, some of the examples are failing with `EntityAlreadyExists` errors. Please fix.

Reply:
@antonbabenko Fixed most of the examples that were failing with error above. NOTE: `iam-eks-role` can't be applied for some time now so I consider it out of the scope to fix it in this PR.